src/main/scala/ru/gaptrakhmanov/telegram/helper/core/package.scala

```diff
@@ -0,0 +1,11 @@
+package ru.gaptrakhmanov.telegram.helper
+
+import java.util.Properties
+import scala.util.Try
+
+package object core {
+  val getProperties: Try[Properties] = {
```

Comment:
Suggestion we could use the MonaError inside of the F[_] instead of using Try[_].